src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -706,6 +761,18 @@ public void PlaceOrganelle()
         }
     }
 
+    public void ResetCamera()
+    {
+        camera.CameraHeight = camera.DefaultCameraHeight;
+        camera.Translation = new Vector3(0, camera.Translation.y, 0);
+        organelleRot = 0;
+    }
+
+    public void TranslateCam(Vector3 dir)
```

Comment:
This should be private?

Reply:
#1719

Reply:
You are adding a new thing in this PR, you can make that added thing private.

Reply:
yes, but this is a general issue all over this file.
Because of that, I created an additional, good first issue issue.